diff --git a/src/_http/index.js b/src/_http/index.js
index 8759f1b8543f2910e4ae4e1cf91a5757b307df50..bcf1c15616d7110f8e706bbf489cc8124861ba30 100644
--- a/src/_http/index.js
+++ b/src/_http/index.js
@@ -19,15 +19,7 @@ function HttpClient(options) {
     options.port = isHttps ? 443 : 80
   }
 
-  // HTTP2 adapter is applicable only if it's NodeJS env and
-  // no fetch API override provided (to preserve backward-compatibility).
-  var useHttp2Adapter = !options.fetch && util.isNodeEnv() && isHttp2Supported()
-
-  this._adapter = useHttp2Adapter
-    ? new (require('./http2Adapter'))({
-        http2SessionIdleTime: options.http2SessionIdleTime,
-      })
-    : new (require('./fetchAdapter'))({
+  this._adapter = new (require('./fetchAdapter'))({
         isHttps: isHttps,
         fetch: options.fetch,
         keepAlive: options.keepAlive,